tests/test_lib_smsprovider.py

```diff
@@ -645,6 +694,13 @@ def setUp(self):
         self.provider = create_sms_instance(identifier=identifier)
         self.assertEqual(type(self.provider), SmppSMSProvider)
 
+        identifier_regexp = "myregexpSmppGW"
```

Comment:
And this part into `test_04_send_sms_regexp_success`
Also You don't need the asserts here, by now we should have established that creating an SMSProvider should work.